examples/testing-network-shim-app/.env

```diff
@@ -1 +1,3 @@
-SKIP_PREFLIGHT_CHECK=true
\ No newline at end of file
+SKIP_PREFLIGHT_CHECK=true
+DISABLE_ESLINT_PLUGIN=true
+EXTEND_ESLINT=true
```

Comment:
`react-scripts` wasn't starting, so overriding their ESLint things to allow the App to start even if there are violations. It's an example app, so as long as the tests pass I don't really care about if there are lint errors...